pyvcontrol/viData.py

```diff
@@ -417,6 +421,54 @@ def __fromraw__(self,value):
     def value(self):
         return self.OnOff[int.from_bytes(self, 'big')]
 
+
+# holder type for Energy
+Energy = namedtuple('Energy', 'byte1 day year week heating_energy heating_electrical_energy water_energy water_electrical_energy, cooling_energy cooling_electrical_energy total_energy total_eletrical_energy')
+
+class viDataEnergy(viData):
+    #Energy-Type ... Return from Function-Call B800
+    # see https://github.com/openv/openv/issues/480#issuecomment-712235475
+    # see https://github.com/openv/openv/wiki/FunctionCalls-WO1C
+    unit={'code':'F_E','description': 'returns Named Tuple with enery Data','unit':''},      
+
+    def __init__(self, value=b'\x00\x00',len=16):
+        #sets int representation based on input value
+        self.len=len  #length in bytes
+        super().__init__(value)
+
+    def __fromtype__(self,value):
+        raise viDataException(f'Value not setable from Type')
+
+    @property
+    def value(self) -> Energy:
+        # decode the Result Record an return a named tuple
+        # see links above
+
+        # example 02 02 16 09 92 03 aa 00 99 00 2d 00 00 00 00 00
+        raw = unpack('<4B6H', self)        
+        
+        res = Energy(
+            raw[0],raw[1],2000+raw[2],raw[3],
+            raw[4]*0.1, raw[5]*0.1, #heating
+            raw[6]*0.1, raw[7]*0.1, #water
+            raw[8]*0.1, raw[9]*0.1, #cooling ??? 
+            raw[4]*0.1+raw[6]*0.1+raw[8]*0.1, raw[5]*0.1+raw[7]*0.1+raw[9]*0.1 #total
+        )
+
+        return res
+        
+    @property
```

Comment:
Diese Funktion gehört m.E. thematisch zu viTools, nicht zu viData, da sie nur in dem speziellen Kontext verwendet wird.
viTools verweist auf viData, viData wiederum auf viTools (quasi zirkuläre Abhängigkeit)